CoroutineTemplate/app/src/main/res/values/colors_pallete.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <!-- Declaring colors with their real name from the style guide we get -->
+    <color name="light_blue">#3F51B5</color>
+    <color name="dark_blue">#303F9F</color>
+    <color name="pink">#FF4081</color>
```

Comment:
Can we set all of them with the real color name from https://www.htmlcsscolor.com/hex/3F51B5 ?

Reply:
Updated 🤖